ros2multicast/setup.cfg

```diff
@@ -26,6 +26,10 @@ tests_require =
   pytest-cov
 zip_safe = true
 
+[options.data_files]
```

Comment:
why is `ros2multicast` different from all the other changes here where `data_files` are installed via `setup.py`?

Reply:
Because the package specifies all its arguments in a `setup.cfg` file. Afaik that is the more modern approach.

Reply:
sorry for the noise, this change requires setuptools v42 and does not work with setuptools distributed via debians (v39).